doc/changelog/11-standard-library/17787-app_inj_pivot.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  lemmas :g:`app_eq_cons`, :g:`app_inj_pivot` and :g:`rev_inj`
+  (`#17787 <https://github.com/coq/coq/pull/17787>`_,
+  by Stefan Haan and Olivier Laurent).
```

Comment:
```suggestion
  by Stefan Haan, with help of Olivier Laurent).
```
I think this is the [recommended format](https://github.com/coq/coq/tree/master/doc/changelog#how-to-add-an-entry).